app/src/main/java/org/luxoft/sdl_core/BleLocalSocketReader.java

```diff
@@ -98,7 +102,13 @@ public void run() {
                     String stringified_data = new String(buffer);
                     Log.d(TAG, "Receive data from socket = "
                             + stringified_data);
-                    mCallback.OnMessageReceived(buffer);
+                    String new_string = new String("message " + counter);
+                    counter += 1;
+
+                    byte[] new_buffer;
+                    new_buffer = new_string.getBytes();
```

Comment:
Is it possible to merge these two lines into a single one?

Reply:
done in 1537bc4